aip/general/0203.md

```diff
@@ -24,10 +24,12 @@ field behavior, such as a field being required or immutable.
 RecognitionAudio audio = 2 [(google.api.field_behavior) = REQUIRED];
 ```
 
-- APIs **must** apply the `google.api.field_behavior` annotation on every field.
+- APIs **must** apply the `google.api.field_behavior` annotation on every field
+  on a message or sub-message used in a request.
```

Comment:
Thank you for adding that extra scope to the guidance.

Reply:
of course! missed it in my first pass.